cmd/oidc-example-op/server.go

```diff
@@ -37,7 +37,7 @@ const loginPage = `<!DOCTYPE html>
 			<p>Subject: <input type="text" name="subject" value="auser" required size="15"></p>
 			<p>Granted Scopes (space delimited): <input type="text" name="scopes" value="{{ .acr }}" size="15"></p>
 			<p>ACR: <input type="text" name="acr" size="15"></p>
-			<p>AMR: <input type="text" name="amr" value="{{ .acr }}" size="15"></p>
+			<p>AMR (comma delimited): <input type="text" name="amr" value="{{ .acr }}" size="15"></p>
```

Comment:
It looks like `value="{{ .acr }}"` is duplicated on this line and line 38, possibly a copy & paste error.

Reply:
100% a copy and paste error, fixed!